app/api/report.py

```diff
@@ -204,8 +204,29 @@ def get_manager_weekly_list(weekly_id=None):
         }, upsert=False)
 
         return jsonify(ret)
+    
+    
 
-
+@bp.route("/overall_reviewes/<string:user_id>", methods=["GET"])
+@jwt_required
+def overall_reviewes(user_id):
+    docs = mongo.db.reports.find({"user":user_id})
+    user = mongo.db.users.find_one({"_id":ObjectId(user_id)})
+    weights = user['managers']
+    manager_weight = weights[0]
+    weight = manager_weight['weight']  
+    docs = [serialize_doc(doc) for doc in docs]
+    sum=0
+    for detail in docs:   

```

Comment:
don't use for loop. use list comprehension

Reply:
ok